engine/CMakeLists.txt

```diff
@@ -69,6 +80,8 @@ INCLUDE(CheckTypeSize)
 INCLUDE(CMakePushCheckState)
 SET(SIZEOF_VOID_P CMAKE_SIZEOF_VOID_P)
 
+SET(CMAKE_FIND_FRAMEWORK "LAST")
```

Comment:
we probably should set this in the root CMakeLists.txt

Reply:
I actually set `CMAKE_FIND_FRAMEWORK` a total of six times in this same file, flipping back and forth between `"LAST"` and `"FIRST"`, for reasons related to OpenGL on macOS. It's weird, but it seems to work.

Reply:
yeah - I came across that when I was trying to figure out the MacOS stuff too. Fair enough.